fix: set upgrade timeout after `startFlushUpgradeHandshake in `chanUpgradeAck` (#1040)
cosmos/ibc: spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

    // if we are already at flushing, then no state changes occur

>     // upgrade is blocked on this channelEnd from progressing until flush completes on both ends

Reviewer: ```suggestion
    // upgrade is blocked on this channelEnd from progressing until flush completes on its end
```
